src/AWS/Lambda/Events/ApiGateway/ProxyResponse.hs

```diff
@@ -88,6 +93,14 @@ import           Network.HTTP.Types.Status (Status (..), accepted202,
                                             unprocessableEntity422,
                                             unsupportedMediaType415,
                                             upgradeRequired426, useProxy305)
+import           GHC.Generics               (Generic (..))
+
+-- This function is available in Data.Functor as of base 4.11, but we define it
+-- here for compatibility.
+(<&>) :: Functor f => f a -> (a -> b) -> f b
+(<&>) x f = f <$> x
```

Comment:
Could we raise the `base` lower bound? `base-4.11` is GHC 8.4.1 (Mar 2018).

Reply:
I like that this lib is very broadly compatible.  Certainly at some point it'll make sense to start dropping support of older versions--but this seems like a pretty minor convenience here to do so :)